content/en/blog/2023/sig-release-spotlight/sig-release-spotlight.md

```diff
@@ -0,0 +1,80 @@
+![diagram-2](/content/en/blog/2023/sig-release-spotlight/sig-release-meetings.png)
+
+**5. What is the Release Shadow Program and how is it different from other shadow programs included in various other SIGs?**
+
+The Release Shadow Program offers a chance for interested individuals to shadow experienced members of the Release Team throughout a Kubernetes release cycle. This is a unique chance to see all the hard work that a Kubernetes release requires across sub-teams. A lot of people think that all we do is cut a release every three months, but that’s just the top of the iceberg.
+
+Our program typically aligns with a specific Kubernetes release cycle, which has a predictable timeline of approximately three months. While this program doesn’t involve writing new Kubernetes features, it still requires a high sense of responsibility since the Release Team is the last step between a new release and thousands of contributors, so it’s a great opportunity to learn a lot about modern software development cycles at an accelerated pace.
+
+**6. What are the qualifications that you generally look for in a person to volunteer as a release shadow/release lead for the next Kubernetes release?**
+
+While all the roles require some degree of technical ability, some require more hands-on experience with Go and familiarity with the Kubernetes API while others require people who are good at communicating technical content in a clear and concise way. It’s important to mention that we value enthusiasm and commitment over technical expertise from day 1. If you have the right attitude and show us that you enjoy working with Kubernetes and or/release engineering, even if it’s only through a personal project that you put together in your spare time, the team will make sure to guide you. Being a self-starter and not being afraid to ask questions can take you a long way in our team.
+
+**7. What will you suggest to someone who has got rejected from being a part of the Release Shadow Program several times?**
+
+Keep applying.
+
+With every release cycle we have had an exponential growth in the number of applicants, so it gets harder to be selected, which can be discouraging, but please know that getting rejected doesn’t mean you’re not talented. It’s just practically impossible to accept every applicant, however here's an alternative that we suggest:
+
+*Start attending our weekly Kubernetes SIG Release meetings to introduce yourself and get familiar with the team and the projects we are working on.*
+
+The Release Team is the only way to join SIG Release, but we are always looking for more hands to help. Again, in addition to certain technical ability, the most sought after trait that we look for is people we can trust, and that requires time.
```

Comment:
Is this correct though? I think that the Release Team takes on some of the responsibility of this specific subproject, but it's different from being the only way to join SIG Release.

Reply:
I had the same doubt too. I'll reconfirm this thing.